tests/test_generate_file.py

```diff
@@ -133,7 +133,7 @@ def test_generate_file_verbose_template_syntax_error(env, expected_msg):
             context={'syntax_error': 'syntax_error'},
             env=env,
         )
-    assert str(exception.value) == expected_msg
+    assert expected_msg_regex.match(str(exception.value)) is not None
```

Comment:
```suggestion
    assert expected_msg_regex.match(str(exception.value))
```
There is no need to make sure if `re.match()` is `not None` since any other value than `[0,False,None]` is considering as `True`.